.github/workflows/auto-review-and-pr-summary.yml

```
@@ -0,0 +1,102 @@
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_REPOSITORY: ${{ github.repository }}
+          PR_NUMBER: ${{ github.event.pull_request.number }}
+
+      - name: Generate PR body using Gemini API
+        id: generate_pr_body
+        run: |
+          SUMMARY="${{ steps.wait_for_comment.outputs.summary }}"
+          
+          # PRテンプレートを読み込む
+          PR_TEMPLATE=$(cat .github/PULL_REQUEST_TEMPLATE.md)
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)